components/dash-core-components/src/components/Input.react.js

```diff
@@ -121,6 +126,23 @@ export default class Input extends PureComponent {
         } else {
             this.props.setProps({value});
         }
+        this.setState({pendingEvent: undefined});
+    }
+
+    debounceEvent(time = 0.5) {
+        const {value} = this.input.current;
+        const MILLISECONDS = 1000;
```

Comment:
Move to global or use 1000 directly.

Reply:
I would prefer to use 1000 directly, but there is a linter rule preventing that :( 
Since this number is only relevant to this function, I'm not sure it makes sense to move it higher.